tests/sentry/grouping/test_enhancer.py

```diff
@@ -46,12 +49,21 @@ def test_basic_parsing(insta_snapshot, version):
     )
     enhancement.version = version
 
-    dumped = enhancement.dumps()
     insta_snapshot(dump_obj(enhancement))
+
+    dumped = enhancement.dumps()
     assert Enhancements.loads(dumped).dumps() == dumped
     assert Enhancements.loads(dumped)._to_config_structure() == enhancement._to_config_structure()
     assert isinstance(dumped, str)
 
+    with override_options({"enhancers.use-zstd": True}):
```

Comment:
I encourage you also test that it is actually using the method you intend to. Not a hard requirement.